ui/src/pages/publications/[id]/versions/[versionId].tsx

```diff
@@ -205,27 +205,28 @@ const Publication: Types.NextPage<Props> = (props): React.ReactElement => {
 
     const peerReviews = linkedFrom.filter((link) => link.type === 'PEER_REVIEW') || [];
 
-    // problems this publication is linked to
-    const parentProblems = linkedTo.filter((link) => link.type === 'PROBLEM') || [];
+    // Publications this publication is linked to (only shown on problems)
+    const parentPublications = publicationVersion?.publication.type === 'PROBLEM' ? linkedTo : [];
```

Comment:
Not sure if PEER REVIEWS should be included, may need to check that with PO when he's back, until then, due to lack of info about this, I think we can move this ticket forward.